components/project/post/SkillsPage.tsx

```diff
@@ -39,7 +39,7 @@ export default ({ project }): JSX.Element => {
         <Typography className={classes.title}>
           <span style={{ fontWeight: 'bold' }}>{`Skills / Tools`}</span>
         </Typography>
-        <ArrayForm label="Skills" items={skills} setItems={setSkills} />
+        <ArrayForm label="Skills" items={skills} setItems={setSkills} options={skills} />
```

Comment:
- need to add options prop in line 26, then pass that into ArrayForm's options
- modify pages/project/post to fetch from backend (I did teamSize and duration choices for example)